WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderUtils.java

```diff
@@ -94,6 +95,21 @@ public static String getLongLikeLabelText(Context context, int numLikes, boolean
     }
 
     /*
+     * short like text ("1 like," "5 likes," etc.)
+     */
+    public static String getShortLikeLabelText(Context context, int numLikes) {
```

Comment:
I wasn't aware of [plurals](http://developer.android.com/guide/topics/resources/string-resource.html#Plurals) until now! I don't see us using plurals anywhere at the moment - do we know whether we can integrate them into our translation process?